src/torchcodec/decoders/_core/VideoDecoder.h

```diff
@@ -308,7 +308,7 @@ class VideoDecoder {
     // The desired position of the cursor in the stream. We send frames >=
     // this pts to the user when they request a frame.
     // We set this field if the user requested a seek.
```

Comment:
Is this comment stale now?

Alternative change would be to not initialize this to 0.